### ISSUE TITLE
mdMenu: Simple Dropdown Example not accessible from keyboard

### ISSUE THREAD

**Issue by marcysutton** [angular/material#4923]
When I use the keyboard to TAB onto the first mdMenu example, I can only activate it with the mouse. It should be operable using the SPACE key and/or ENTER key.

Bug visible on HEAD and 0.11.1. It last worked correctly in 0.10.1.

**Comment by rschmukler** [comment]
@marcysutton this is because we are using the `ng-mouseenter` event on that menu, which does not fire on enter / space. This is potentially a confusing first demo, but unless we wan't to overload the `ng-mouseenter` directive I think this is behaving as intended? Let me know your thoughts

**Comment by marcysutton** [comment]
That doesn't matter to a keyboard user. It still has to work for all users, no matter what the demo is intending to show.

**Comment by rschmukler** [comment]
@marcysutton perhaps it's an error with `ng-mouseenter` then and this issue should be directed to angular itself?

**Comment by marcysutton** [comment]
Just to add some more notes here...we shouldn't be opening menus on `ng-mouseenter`, or `ng-focus` for that matter. It forces a user into a menu, which is bad UX. Recommended the demo be changed to use `ng-click`, which is a more accessible and desirable user experience for this component..

**Comment by rschmukler** [comment]
Sounds good!

**Comment by rschmukler** [comment]
closed by f9d5cf8ed20551ae7fc5f8b4f21407db7fc9bdbc
